lib/superstore/connection.rb

```diff
@@ -9,10 +9,8 @@ def adapter
 
       def adapter_class
         case config[:adapter]
-        when 'jsonb'
+        when nil, 'jsonb'
```

Comment:
That's what I'm thinking. However, I do hope to get this into Rails, which means mysql.